variables.tf

```diff
@@ -6,33 +6,41 @@ variable "location" {
 variable "env" {
   type = string
 }
+variable "product" {
+  type = string
+}
 variable "tags" {
   type        = map(string)
   description = "Runbook Tags"
 }
 ## Azure Automation
-variable "automation_account_name" {
-  type        = string
-  description = "automation account name"
+variable "auto_acc_runbooks" {
+  default = []
```

Comment:
Can you specify the type format and example please?

Reply:
Type specified and example updated